### PR TITLE
fix fschat docker and workflow

### PR DESCRIPTION
**Pull request by jq460494839**
修改fschat工作流，适配dockerfile
SentencePiece 必须安装：
Traceback (most recent call last):
  File "/opt/nlp_data/FastChat/fastchat/train/train.py", line 317, in <module>
    train()
  File "/opt/nlp_data/FastChat/fastchat/train/train.py", line 283, in train
    tokenizer = transformers.AutoTokenizer.from_pretrained(
  File "/root/miniconda/envs/torch_npu/lib/python3.9/site-packages/transformers/models/auto/tokenization_auto.py", line 768, in from_pretrained
    return tokenizer_class.from_pretrained(pretrained_model_name_or_path, *inputs, **kwargs)
  File "/root/miniconda/envs/torch_npu/lib/python3.9/site-packages/transformers/utils/import_utils.py", line 1259, in __getattribute__
    requires_backends(cls, cls._backends)
  File "/root/miniconda/envs/torch_npu/lib/python3.9/site-packages/transformers/utils/import_utils.py", line 1247, in requires_backends
    raise ImportError("".join(failed))
ImportError: 
LlamaTokenizer requires the SentencePiece library but it was not found in your environment. Checkout the instructions on the
installation page of its repo: https://github.com/google/sentencepiece#installation and follow the ones
that match your environment. Please note that you may need to restart your runtime after installation.

### CODE REVIEW

File: docker/fschat/Dockerfile [hunk 1]
```diff
@@ -8,6 +8,8 @@ RUN apt update && \
     libsndfile1 && \
     rm -rf /var/lib/apt/lists/*
 
+RUN git clone https://github.com/npu-ci/llm-tool-ci.git
```
Comment (zhangsibo1129) [inline docker/fschat/Dockerfile:11]: 这个是fschat镜像，为何要克隆llm-tool-ci 呢

File: docker/fschat/Dockerfile [hunk 2]
```diff
@@ -16,3 +18,4 @@ ENV PIP_SOURCE_URL=https://pypi.tuna.tsinghua.edu.cn/simple
 
 RUN cd FastChat && \
     pip install --no-cache-dir -e ".[model_worker,webui]" -i ${PIP_SOURCE_URL}
+    pip install SentencePiece -i ${PIP_SOURCE_URL}
```
Comment (zhangsibo1129) [inline docker/fschat/Dockerfile:21]: sentencepiece 我看在model_worker依赖里面已经有了，而且我们不需要安装 webui依赖吧

**Comment by jq460494839** [comment]
应该把Rust的安装从base的Dockerfile放到transformers里

**Comment by zhangsibo1129** [comment]
> 应该把Rust的安装从base的Dockerfile放到transformers里
可以啊

**Pull request merged**